components/logging/org.wso2.carbon.logging.service/src/main/java/org/wso2/carbon/logging/service/util/LoggingUtil.java

```diff
@@ -442,7 +442,7 @@ public static void restoreDefaults() throws Exception {
         } catch (FileNotFoundException e) {
             String msg = "Cannot restore default logging configuration."
                     + " log4j.properties file not found in the classpath";
```

Comment:
Shall we fix the grammar issue as to ;
" log4j.properties file **is** not found in the **classpath.**"

Reply:
Fixed, thanks for pointing out.